libs/openFrameworks/gl/ofTexture.h

```diff
@@ -177,13 +177,12 @@ class ofTextureData {
 		glInternalFormat = GL_RGB8;
 		textureTarget = GL_TEXTURE_RECTANGLE_ARB;
 #elif defined(TARGET_OPENGLES)
-		if(ofGLESVersionFromGL() >= 300) {
-			glInternalFormat = GL_RGB16F;
-			textureTarget = GL_TEXTURE_2D;
-		} else {
-			glInternalFormat = GL_RGB;
-			textureTarget = GL_TEXTURE_2D;
-		}
+	#if defined(GL_RGB16F)
+		glInternalFormat = GL_RGB16F;
+	#else
+		glInternalFormat = GL_RGB;
+	#endif
```

Comment:
Feels like this would maybe be a safer change as it keeps the original logic, but also checks for the define

```
		if(ofGLESVersionFromGL() >= 300) {
				#if defined(GL_RGB16F)
						glInternalFormat = GL_RGB16F;
				#else
						glInternalFormat = GL_RGB;
				#endif
			textureTarget = GL_TEXTURE_2D;
		} else {
			glInternalFormat = GL_RGB;
			textureTarget = GL_TEXTURE_2D;
		}

Reply:
yeah - I think I was just imagining possible regressions where it might now default to GL_RGB16F instead of GL_RGB - one is a floating point texture and the other is unsigned char. 

So that could mean some shaders might behave a little differently - curious if that would be very noticeable or not really an issue. 

I think in general bug fixes should correct breaking bugs or existing issues without changing behavior, hence my caution.